spec/support/shared_examples/job_template.rb

```diff
@@ -25,6 +25,13 @@
       expect(described_class.new(api, raw_instance_no_survey).survey_spec).to be_nil
     end
   end
+
+  describe "unlicensed" do
```

Comment:
@bdunne - This looks great.  It would be great to have some testing around the middleware as well.